test/process.test.js

```diff
@@ -97,3 +98,62 @@ test('process | pending payouts', async (t) => {
 
   t.deepEqual(result, { success: true })
 })
+
+test('process | pending payouts only partial paid', async (t) => {
+  // Ilp says it didn't complete 1000 worth of the payouts
+  t.context.deps.ilp.sendIlpPayment.resolves({ success: false, remainingAmount: 1000 })
+
+  const record = {
+    body: JSON.stringify({
+      maintainerId: 'aaaaaaaaaaaa'
+    })
+  }
+  const { deps } = t.context
+
+  deps.db.getPendingMaintainerPayouts.resolves({
+    payouts: [
+      {
+        id: 'payout-id',
+        amount: 12345
+      },
+      {
+        id: 'payout-id2',
+        amount: 54321
+      }
+    ],
+    ilpPaymentPointer: '$helloworld'
+  })
+
+  const input = { ...deps, record }
+
+  try {
+    await Process.process(input)
+  } catch (e) {
+    t.deepEqual(e.message, 'For some reason ILP sender failed')
+  }
```

Comment:
I think this should be a t.throwsAsync so we know that it throws (I think if it doesn't throw, ava won't care that the t.deepEqual was never called).